velox/exec/TableScan.cpp

```diff
@@ -18,7 +18,7 @@
 #include "velox/exec/Task.h"
 #include "velox/expression/Expr.h"
 
-DEFINE_int32(split_preload_per_driver, 0, "Prefetch split metadata");
+DEFINE_int32(split_preload_per_driver, 2, "Prefetch split metadata");
```

Comment:
Hi @xiaoxmeng 
Does this PR resolve split prefetch issue mentioned at #4010 ?

Reply:
We meet similar issue in Gluten https://github.com/oap-project/gluten/issues/1124, and curious about why this flag enable again?